app/posts/teacher-success/2025-04-25-testing-support-comms-with-trainees.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Testing supportive comms with trainee teachers
+description: Detailing email comms we sent to trainees to test their engagement with supportive content from Department for Education
+date: 2025-04-25
+tags:
+  - withdrawals
+  - trainees
+  - emails
+  - comms
+---
+
+We wanted to test engagement from trainees with supportive email comms from Department for Education.
+
+[View the full comms testing report](https://educationgovuk.sharepoint.com/:p:/r/sites/TeacherServices/Shared%20Documents/TWD%20-%20Teaching%20Vacancies/TEACHER%20SUCCESS%20SERVICE%20ALPHA%20-%202025/Design%20and%20content/Comms%20testing%20results.pptx?d=w181456ee8022433ab92d68bd2cb6c458&csf=1&web=1&e=35y4Ln).
```

Comment:
Can you change this to a PDF and upload it to the design history site (in your post's images directory)? External users cannot access SharePoint, so we try to avoid adding links like this.

Reply:
Thanks Simon - have tried to add as a PDF but don't think I've done it right!

Reply:
I've pushed a change to your pull request, so the PDF link should now work. It needed to be a link rather than an image reference.